modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->centerTargetOnScreen();
+    }
+        
+    return 0;
+}
+
+int setSelectedBrowser(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::setSelectedBrowser");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+        
+    module->setSelectedBrowser(id);
+        
+    return 0;
+}
+
+int createTargetBrowserPair(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::createTargetBrowserPair");
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    int noOfPairs = module->nPairs();
+    std::string nameBrowser = "Sky Browser " + std::to_string(noOfPairs);
+    std::string nameTarget = "Sky Target " + std::to_string(noOfPairs);
+    std::string idBrowser = "SkyBrowser" + std::to_string(noOfPairs);
+    std::string idTarget = "SkyTarget" + std::to_string(noOfPairs);
+    glm::vec3 positionBrowser = { -1.0f, -0.5f, -2.1f };
+    std::string guiPath = "/SkyBrowser";
+    std::string url = "https://data.openspaceproject.com/dist/skybrowser/page/";
+
+    const std::string browser = "{"
+        "Identifier = '" + idBrowser + "',"
+        "Type = 'ScreenSpaceSkyBrowser',"
+        "Name = '" + nameBrowser + "',"
+        "Url = '" + url + "',"
+        "FaceCamera = false,"
+        "CartesianPosition = " + ghoul::to_string(positionBrowser) +
+        "}";
+    const std::string target = "{"
+        "Identifier = '" + idTarget + "',"
+        "Type = 'ScreenSpaceSkyTarget',"
+        "Name = '" + nameTarget + "',"
+        "FaceCamera = false,"
+        "}";
+
+    global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + browser + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + target + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.skybrowser.addPairToSkyBrowserModule('" + idTarget + "','"
+        + idBrowser + "');",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.skybrowser.setSelectedBrowser('" + idBrowser + "');",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    return 0;
+}
+
+int removeTargetBrowserPair(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::removeTargetBrowserPair");
+    std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* found = module->getPair(id);
+    if (found) {
```

Comment:
see comment above, about chekcing and returning a lua error if it's not found

here it even adds a benefit in tat the indentation will be reduces for all of the lines inside the current if-statement :)